### PR TITLE
Replace swsssdk with swsscommon in bgpmon.py

### PR DESCRIPTION
**Pull request by liuh-80**
#### Why I did it
Update bgpmon.py from py-swsssdk to swsscommon

#### How I did it
Change bgpmon.py to use swsscommon.

#### How to verify it
Pass all E2E test case

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205

#### Description for the changelog
Update bgpmon.py from py-swsssdk to swsscommon

### CODE REVIEW

**Comment by liuh-80** [comment]
Still under discussion about how to improve hmset method.

**Comment by liuh-80** [comment]
This PR related with another PR which will improve hmset performance, but not depends on it:
https://github.com/Azure/sonic-swss-common/pull/647/

File: src/sonic-bgpcfgd/bgpmon/bgpmon.py [hunk 1]
```diff
@@ -106,11 +104,10 @@ def flush_pipe(self, data):
         for key, value in data.items():
             if value is None:
                 # delete case
-                self.pipe.delete(key)
+                self.db.delete(self.db.STATE_DB, key)
             else:
                 # Add or Modify case
-                self.pipe.hmset(key, value)
-        self.pipe.execute()
+                self.db.hmset(self.db.STATE_DB, key, value)
```
Comment (liuh-80) [inline src/sonic-bgpcfgd/bgpmon/bgpmon.py:110]: Will improve this code with pipe

Comment (liuh-80) [reply src/sonic-bgpcfgd/bgpmon/bgpmon.py]: Fixed, now this PR depends on https://github.com/Azure/sonic-swss-common/pull/647
We need that PR finish and update submodule first.

Comment (liuh-80) [reply src/sonic-bgpcfgd/bgpmon/bgpmon.py]: sonic-swss-common PR merged, create another PR to merge submodule: https://github.com/Azure/sonic-buildimage/pull/11470

After that, this PR can pass UT.

**Review by qiluo-msft — APPROVED** [review]

**Comment by liuh-80** [comment]
Will delay merge this PR because a resource leak issue in swss-common.

**Comment by liuh-80** [comment]
Wait for this PR merge first: https://github.com/Azure/sonic-buildimage/pull/11524 which will fix memory leak issue.

**Pull request merged**